ledger/participant-integration-api/src/main/scala/platform/index/BuffersUpdater.scala

```diff
@@ -57,9 +67,15 @@ private[index] class BuffersUpdater(
           randomFactor = 0.0,
         )
       )(() => subscribeToTransactionLogUpdates(updaterIndex.get))
-      .map { case ((offset, eventSequentialId), update) =>
-        updateCaches(offset, update)
-        updaterIndex.set(Some(offset -> eventSequentialId))
+      .async
+      .mapAsync(1) { case ((offset, eventSequentialId), update) =>
```

Comment:
Is it so that mapAsync 1 is very much mandatory due to the nature of the updates? If yes,c an we explain that in a comment so that this is not accidentally parametrised / increased?